### ISSUE TITLE
FileSystemException: FileSystemException: Failed to decode data using encoding 'utf-8',

### ISSUE THREAD

**Issue by williamtong0228** [flutter/flutter#55629]
## Steps to Reproduce

1. Fresh install of Flutter following official website's tutorial
2. But I installed it on an external ssd
3. And I am using Macbook pro 2018
4. No matter which way I use, VS code, Android Studio, command line. I always get the same result

## Logs

flutter --no-color create --offline --template app --description A new Flutter application. --org com.wingluntong --project-name flutterappnoway --ios-language objc flutter_app_noway

## exception

FileSystemException: FileSystemException: Failed to decode data using encoding 'utf-8', path = '/Volumes/ExtremeSSD/flutter/packages/flutter_tools/templates/app/ios.tmpl/Runner/._Info.plist.tmpl'

```
#0      _File._tryDecode (dart:io/file_impl.dart:574:7)
#1      _File.readAsStringSync (dart:io/file_impl.dart:594:7)
#2      ForwardingFile.readAsStringSync (package:file/src/forwarding/forwarding_file.dart:103:16)
#3      Template.render.<anonymous closure> (package:flutter_tools/src/template.dart:184:52)
#4      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:379:8)
#5      Template.render (package:flutter_tools/src/template.dart:132:24)
#6      CreateCommand._renderTemplate (package:flutter_tools/src/commands/create.dart:667:21)
#7      CreateCommand._generateApp (package:flutter_tools/src/commands/create.dart:578:23)
#8      CreateCommand.runCommand (package:flutter_tools/src/commands/create.dart:429:37)
<asynchronous suspension>
#9      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:615:18)
#10     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:73:64)
#11     _rootRunUnary (dart:async/zone.dart:1134:38)
#12     _CustomZone.runUnary (dart:async/zone.dart:1031:19)
#13     _FutureListener.handleValue (dart:async/future_impl.dart:139:18)
#14     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:680:45)
#15     Future._propagateToListeners (dart:async/future_impl.dart:709:32)
#16     Future._completeWithValue (dart:async/future_impl.dart:524:5)
#17     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:554:7)
#18     _rootRun (dart:async/zone.dart:1126:13)
#19     _CustomZone.run (dart:async/zone.dart:1023:19)
#20     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
#21     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:965:23)
#22     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#23     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#24     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#25     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:175:5)
```

## flutter doctor

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.5 19F62f, locale en-HK)
    • Flutter version 1.12.13+hotfix.9 at /Volumes/ExtremeSSD/flutter
    • Framework revision f139b11009 (4 weeks ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/william/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.44.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.9.1

[✓] Connected device (1 available)
    • iPhone 8 • 60FA21B1-502F-4DCA-886D-FA78A8B09B3C • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)

• No issues found!
```

**Comment by jonahwilliams** [comment]
Interestingly there is not file `flutter_tools/templates/app/ios.tmpl/Runner/._Info.plist.tmpl` in the flutter repo. Can you confirm that your SDK is unmodified by running `git status` in `/Volumes/ExtremeSSD/flutter`?

**Comment by williamtong0228** [comment]
Hi, it says that it's 
not a git repository(or any of the parent directories):.git

When I was installing flutter. It could not be installed onto the external SSD which is exFat format. So I followed someone's suggestion and changed the temp folder to my the system $tmp folder. 

The bundle I downloaded was the flutter_macos_v1.12.13+hotfix.9-stable.zip
I can create project on my old mac mini. Thank you.
